### DISCUSSION TITLE
Cannot start new world - Premade world not found

### DISCUSSION THREAD

**Discussion by CyGor84** [Devidian/docker-spaceengineers#55] [Q&A]
Hello,

I'm trying to start a server with docker, but everytime I try to start it, the console on my Dockge tell's me:

space-engineers-server  | 2023-12-18 21:40:51.122: **Cannot start new world - Premade world not found** 
space-engineers-server  | -----------------------------------END GAME----------------------------------
space-engineers-server  | -----------------------------------BYE !!!!----------------------------------
space-engineers-server exited with code 0

Where can I set the new world, or transfer a world from other instance?

**Comment by Devidian** [comment]
Hi,

You have to create a world first with the dedicated server tool. Then upload it to your server. 
I have no detailed instruction right now as i am at work. If you need some please let me know, but i cant tell when i have time to add it.

**Comment by trevorgolden** [comment]
First off, sorry to necro this discussion almost 2 months later, but I am having the same problem.

- Instances
  - GreenStation
    - cache
    - Mods
    - Saves
      - GreenStation
        - BaseAsteroid0.vx2
        - BaseAsteroid1.vx2
        - BaseAsteroid2.vx2
        - Sandbox.sbc
        - SANDBOX_0_0_0_.sbs
        - SANDBOX_0_0_0_.sbsB5
        - Sandbox_config.sbc
        - thumb.jpg
    - temp
    - SpaceEngineers-Dedicated.cfg
    - SpaceEngineersDedicated.log
    
    
This is my file system inside of the Instances folder, what am I missing? 

-----------------------------CURRENT CONFIGURATION---------------------------
GAME_DIR=/appdata/space-engineers/SpaceEngineersDedicated
CONFIG_PATH=/appdata/space-engineers/instances/GreenStation/SpaceEngineers-Dedicated.cfg
INSTANCE_IP=172.20.0.2
CURRENT_IP=172.20.0.2
CURRENT_WORLDNAME=
SAVE_PATH=Z:\\appdata\\space-engineers\\instances\\GreenStation\\Saves\\
wine-8.0
----------------------------------START GAME---------------------------------
2024-02-03 00:21:09.778: Cannot start new world - Premade world not found

**Reply by Devidian** [reply]
My relocation is almost done but i've still much work to do until i can focus on help here sorry. I Also will re-investigate the repo and maybe upgrade wine. Then i will add a detailled instruction how to setup an instance correctly.

**Comment by cake314** [comment]
Hello. I am having the same issue as above using the same folder structure. I am not sure how to get the image to accept the pre-made world from the dedicated server app.

**Comment by realM4C** [comment]
Hello,

first create a new world and start it once with the Deditacted Server Tool.

Upload everything and renaming the folder 
`instance/<instance_name>/Saves/StarSystem 05-25-2025 19-44-13`
to
`instance/<instance_name>/Saves/StarSystem`
get it working for me.

Verify that in the "SpaceEngineers-Dedicated.cfg" the WorldName is set correct:
`<WorldName>StarSystem</WorldName>`

May someone helps this.

**Reply by MasonTomkins11** [reply]
This worked! thank you, Essentially whatever you set the world name to in the initial set-up before transferring you just have to modify the save file to match!

**Comment by CobaltPhoenixSRD** [comment]
I have built the world and imported it but i am still getting the premade world not found, i think i have followed everything correctly any suggestions?

I feel like the issue im running into may be related to where the .cfg file is?
